api/v1beta2/integrationtestscenario_types.go

```diff
@@ -103,6 +103,10 @@ type ResolverRef struct {
 	// the chosen resolver.
 	// +required
 	Params []ResolverParameter `json:"params"`
+	// ResolverType defines the type of resolver. It can either be "pipeline" or "pipelinerun"
+	// but defaults to "pipeline" if no value is set
```

Comment:
Defaults to **pipeline** will be happening by our controller I suppose, I don't see a way of setting a default here, if that is even possible.

Reply:
Yup, I think it's more like controller will default to pipeline if this field is not defined.